iceoryx_examples/icedelivery_on_c/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+# Build icedelivery example
+cmake_minimum_required(VERSION 3.5)
+project(example_icedelivery_on_c)
+
+include(GNUInstallDirs)
+
+find_package(iceoryx_posh CONFIG REQUIRED)
+find_package(iceoryx_binding_c CONFIG REQUIRED)
+
+get_target_property(ICEORYX_CXX_STANDARD iceoryx_posh::iceoryx_posh CXX_STANDARD)
+if ( NOT ICEORYX_CXX_STANDARD )
```

Comment:
can we somehow force a C compiler here?
Maybe by setting the language in the `project` call?

Reply:
done by setting the language in the cmake file for the source files to C